Vite gives error while running an application having nuka-carousel

# Bugs and Questions
Vite gives an error while running an application having nuka-carousel

### Describe Your Environment

- What version of nuka-carousel are you using?
- 5.0.13
- What version of React are you using?
- 17.0.2
- What browser are you using?
- Google Chrome 100.0.4896.127

### Describe the Problem

When running my vite React project by using `npm run dev`, I'm getting the following error-

```
X [ERROR] [plugin vite:dep-scan] Failed to resolve entry for package "nuka-carousel". The package 
may have incorrect main/module/exports specified in its package.json: Failed to resolve entry for 
package "nuka-carousel". The package may have incorrect main/module/exports specified in its package.json.
```
Below is the nuka-carousel's package.json entry - 

![image](https://user-images.githubusercontent.com/15645414/165332509-5bb613c0-84bc-42e4-86bf-52f644aadfc1.png)


### Additional Information

On inspection, I found that the module entry is incorrect. The error stops after I manually change the data for the `"module"` key to `"es-v5/index.js"` in nuka-carousel's package.json and re-run my application.

![image](https://user-images.githubusercontent.com/15645414/165335693-2cc0808a-2305-4c9d-bcd2-2a78a4fa34d6.png)

But if I do an npm install later, the error reappears as the package.json gets reset.

--- a/package.json
+++ b/package.json
@@ -3,8 +3,8 @@
   "version": "5.0.13",
   "description": "Pure React Carousel",
   "main": "index.js",
-  "module": "es/index.js",
-  "jsnext:main": "es/index.js",
+  "module": "es-v5/index.js",
+  "jsnext:main": "es-v5/index.js",
   "types": "index.d.ts",
   "engines": {
     "node": ">=12.0.0"

